### PR TITLE
feat: aur, completions, manpages, fix deprecations

### PR DESCRIPTION
**Pull request by caarlos0**
this PR:

- adds aur support
- adds another goreleaser config which supports manpages and completions
- fix some template deprecations

### CODE REVIEW

File: goreleaser.yaml [hunk 1]
```diff
@@ -76,6 +77,14 @@ brews:
     homepage: '{{ .homepage }}'
     description: '{{ .description }}'
 
+aurs:
+  - maintainers: ['{{ .Var.maintainer }}']
+    description: '{{ .Var.description }}'
+    name: '{{ with .Var.aur_project_name }}{{ . }}{{ else }}{{ .ProjectName }}{{ end }}-bin'
+    homepage: '{{ with .Var.homepage }}{{ . }}{{ else }}https://github.com/caarlos0/{{ .ProjectName }}{{ end }}'
```
Comment (muesli) [inline goreleaser.yaml:84]: Should probably change that to `github.com/charmbracelet`.

Comment (caarlos0) [reply goreleaser.yaml:84]: bad copy pasta, good catch

**Review by aymanbagabas — APPROVED** [review]

File: goreleaser-full.yaml [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+      - rpm
+    contents:
+      - src: ./completions/{{ .ProjectName }}.bash
+        dst: /etc/bash_completion.d/{{ .ProjectName }}
+      - src: ./completions/{{ .ProjectName }}.fish
+        dst: /usr/share/fish/completions/{{ .ProjectName }}.fish
+      - src: ./completions/{{ .ProjectName }}.zsh
+        dst: /usr/local/share/zsh/site-functions/_{{ .ProjectName }}
+      - src: ./manpages/{{ .ProjectName }}.1.gz
+        dst: /usr/share/man/man1/{{ .ProjectName }}.1.gz
```
Comment (caarlos0) [inline goreleaser-full.yaml:76]: main change from the other file

File: goreleaser-full.yaml [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+      name: '{{ .Var.brew_commit_author_name }}'
+      email: '{{ .Var.brew_commit_author_email }}'
+    homepage: '{{ .Var.homepage }}'
+    description: '{{ .Var.description }}'
+    install: |-
+      bin.install "{{ .ProjectName }}"
+      bash_completion.install "completions/{{ .ProjectName }}.bash" => "{{ .ProjectName }}"
+      zsh_completion.install "completions/{{ .ProjectName }}.zsh" => "_{{ .ProjectName }}"
+      fish_completion.install "completions/{{ .ProjectName }}.fish"
+      man1.install "manpages/{{ .ProjectName }}.1.gz"
```
Comment (caarlos0) [inline goreleaser-full.yaml:93]: main change from the other file

File: goreleaser-full.yaml [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+      install -Dm644 "./LICENSE.md" "${pkgdir}/usr/share/licenses/{{ .ProjectName }}/LICENSE"
+      # completions
+      mkdir -p "${pkgdir}/usr/share/bash-completion/completions/"
+      mkdir -p "${pkgdir}/usr/share/zsh/site-functions/"
+      mkdir -p "${pkgdir}/usr/share/fish/vendor_completions.d/"
+      install -Dm644 "./completions/{{ .ProjectName }}.bash" "${pkgdir}/usr/share/bash-completion/completions/{{ .ProjectName }}"
+      install -Dm644 "./completions/{{ .ProjectName }}.zsh" "${pkgdir}/usr/share/zsh/site-functions/_{{ .ProjectName }}"
+      install -Dm644 "./completions/{{ .ProjectName }}.fish" "${pkgdir}/usr/share/fish/vendor_completions.d/{{ .ProjectName }}.fish"
+      # man pages
+      install -Dm644 "./manpages/{{ .ProjectName }}.1.gz" "${pkgdir}/usr/share/man/man1/{{ .ProjectName }}.1.gz"
```
Comment (caarlos0) [inline goreleaser-full.yaml:116]: main change from the other file

**Pull request merged**